zwoasi/__init__.py

```diff
@@ -464,6 +470,10 @@ def close(self):
         The destructor will automatically close the camera if it has not already been closed."""
         try:
             _close_camera(self.id)
+        except:
```

Comment:
Is this leftover debugging code or an intended code path?

Reply:
Yeh, we could hit a situation whereby the camera has been disconnected before programatically disconnected?

Reply:
I have removed it for now. I would argue that it would be useful to alert that closing the camera for some reason failed. But would rather this gets merged in.